app/views/pipelines/_card.html.erb

```diff
@@ -1,19 +1,32 @@
 <% jobs_path ||= ''
    run_sample_extraction_path ||= ''
-   run_full_extraction_path ||= '' %>
+   run_full_extraction_path ||= ''
+   type = definition.class.name.gsub(/Definition$/, '').downcase
+   position = type == 'extraction' ? 'first' : 'last' %>
 
 <div class="card">
   <div class='card-body'>
-    <h5 class="card-title"><%= definition.send("#{type}_definition").name %></h5>
+    <h5 class="card-title"><%= definition.name %></h5>
+    <h6 class="card-subtitle">
+      <%= last_edited_by(definition) %>
```

Comment:
There's a fair bit of logic here, would it be good to put it in a partial and test it with a view test?

Reply:
Or maybe just have a view test for this partial

Reply:
Actually I'll make this helpers